src/mjolnir/ferry_connections.cc

```diff
@@ -333,6 +333,7 @@ void ReclassifyFerryConnections(const std::string& ways_file,
     auto bundle = collect_node_edges(node_itr, nodes, edges);
     if (bundle.node.ferry_edge_ && bundle.node.non_ferry_edge_ &&
         GetBestNonFerryClass(bundle.node_edges) > kFerryUpClass &&
+        // TODO(nils): why exactly do we not want to consider short ferries?
```

Comment:
yeah, this doesn't make any sense to me:) also short ferries can provide a huge shortcut, at least in distance

Reply:
well, "huge" shortcut is maybe a wild exaggeration, considering this'll only be important when hierarchy limits are exhausted, i.e. long trip..